README.md

```diff
@@ -4,11 +4,7 @@
 [**What are DataPipes?**](#what-are-datapipes) | [**Beta Usage and Feedback**](#beta-usage-and-feedback) |
```

Comment:
Can you please update the title to something like `# TorchData ((🚨 Warning: Development Paused 🚨)`

Reply:
Done